rotkehlchen/api/v1/schemas.py

```diff
@@ -1447,7 +1430,7 @@ def _transform_btc_or_bch_address(
     return address
 
 
-def _transform_eth_address(
+def _transform_evm_address(
         ethereum_inquirer: EthereumInquirer,
         given_address: str,
 ) -> ChecksumEvmAddress:
```

Comment:
Here is only ethereum that has `ens_lookup`. If you pass an ens name it should raise an error for chains that don't have it right?